src/global/SEOHead.tsx

```diff
@@ -62,6 +62,7 @@ const SEOHead = ({ title, desc, network }: Props) => {
 				content={title}
 			/>
 			<meta
+				// yaha change hoga
```

Comment:
Is this needed?

Reply:
no sir just commented here to remind myself...removing it